core/src/image-generation/image-generation.controller.ts

```diff
@@ -0,0 +1,14 @@
+import {Controller, Get, Param} from "@nestjs/common";
+
+import {ImageGenerationService} from "./image-generation.service";
+
+@Controller("image-gen")
```

Comment:
GraphQL, not Rest please

Reply:
Does this actually need a controller? I meant to ask that. I threw this up for really easy testing, but we'll this be called from the web or just imported by the scrim module and called from within nest?

Reply:
That's a good question, and something that we probably need to explore more in the future. Leave it in for now.